pallets/external-validators/src/lib.rs

```diff
@@ -380,8 +385,13 @@ pub mod pallet {
         ) -> DispatchResult {
             // If more validators than max, take the first n
             let validators = BoundedVec::truncate_from(validators);
-            <ExternalValidators<T>>::put(validators);
+            <ExternalValidators<T>>::put(&validators);
             <ExternalIndex<T>>::put(external_index);
+
```

Comment:
Tests please!

Reply:
Added tests here https://github.com/moondance-labs/tanssi/pull/857/commits/a18edabb89c431835b855cd5829e3139d64382ef and https://github.com/moondance-labs/tanssi/pull/857/commits/e82b9e7b3ff3ac423d3eb7e7f4331db20b4ab483 . Do we want to update any Moonwall or integration tests as well?

Reply:
Maybe the zombie_tanssi_relay_eth_bridge?

Reply:
Reverted the change in [c044d83](https://github.com/moondance-labs/tanssi/pull/857/commits/c044d83fa6eb2d58c83b730303eb883d9756b6fb)